# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/nvi/no-snprintf.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./PORT/clib/snprintf.c.orig	Wed Mar 21 12:42:09 2001
+++ ./PORT/clib/snprintf.c	Wed Mar 21 12:43:28 2001
@@ -1,3 +1,6 @@
+/* This is not needed. - Clifford */
+#if 0
+
 #include <sys/types.h>
 #include <sys/cdefs.h>
 
@@ -52,3 +55,5 @@
 	return (vsprintf(str, fmt, ap));
 #endif
 }
+
+#endif
